core/tests/ts-integration/src/helpers.ts

```diff
@@ -23,7 +24,8 @@ export function getTestContract(name: string): ZkSyncArtifact {
  */
 export function getContractSource(relativePath: string): string {
     const contractPath = `${process.env.ZKSYNC_HOME}/core/tests/ts-integration/contracts/${relativePath}`;
-    const source = fs.readFileSync(contractPath, 'utf8');
+    const filePath = path.resolve(__dirname, contractPath);
```

Comment:
`ZKSYNC_HOME` is an absolute path, so `contractPath` is also absolute and there is no difference between resolving based on CWD on __dirname.

Reply:
> `ZKSYNC_HOME` is an absolute path, so `contractPath` is also absolute and there is no difference between resolving based on CWD on __dirname.

Ok, Thanks